autokeras/hypermodels/basic.py

```diff
@@ -28,9 +28,9 @@ class DenseBlock(block_module.Block):
     """
 
     def __init__(self,
-                 num_layers=None,
-                 use_batchnorm=None,
-                 dropout_rate=None,
+                 num_layers: int = None,
+                 use_batchnorm: bool = None,
+                 dropout_rate: float = None,
```

Comment:
These three args should all be optional. You may refer to other typed blocks.

Reply:
Changed to optional with 55d0c99.